src/Integration.Vsix/SonarLint.vsct

```diff
@@ -20,12 +20,48 @@
   <Include href="Commands\PackageCommands.vsct"/>
   <Include href="SonarLintDaemon\DaemonCommands.vsct"/>
 
+  <Commands package="guidSonarLintPackage">
+
+    <Groups>
+      <Group guid="guidSonarLintPackageCmdSet" id="MyMenuGroup" priority="0x0600">
+        <Parent guid="guidSHLMainMenu" id="IDM_VS_MENU_EXTENSIONS" />
+      </Group>
+      <!-- The sub menu that parents commands. -->
+      <Group guid="guidSonarLintPackageCmdSet" id="SonarLintCommandSubMenu" priority="0x0600">
+        <Parent guid="guidSonarLintPackageCmdSet" id="MyMenuItem" />
+      </Group>
+    </Groups>
+
+    <Menus>
+      <Menu guid="guidSonarLintPackageCmdSet" id="MyMenuItem" type="Menu" priority="0x0600">
+        <Parent guid="guidSonarLintPackageCmdSet" id="MyMenuGroup" />
+        <Strings>
+          <ButtonText>SonarLint</ButtonText>
+        </Strings>
+      </Menu>
+    </Menus>
+
+  </Commands>
+  
   <Symbols>
     <!-- This is the package guid for the package. -->
```

Comment:
I didnt feel comfortable using the old guids as they are currently in use which might make things confusing. Thats why I added to new ones "guidSonatLintPackage" and "guidSonarLintPackageCmdSet"